index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="heading">
+        <a class="logo" href="#"> 
+            John Doe
+        </a>
+        <a href="#">
+            <img class="menu" src="./assets/menu_white_24dp.svg" alt="menu_bar">
+        </a>
+    </header>
+    <div class="job-title">
+        <div class="person-name">
+            <h1> Aku</h1>
+            <h2>Software Developer</h2>
+        </div>
+        <img class="ellipse" src="./assets/Ellipse18.png" alt="">
+
+    </div>
+    <p class="discription">
+        I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hasitate to contact me.
+    </p>
+    <hr>
+    <div class="language-framework">
+        <div>
+            <h6>Language</h6>
+            <ul>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+        </div>
+        <div>
+            <h6>Frameworks</h6>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+        <img class="side-ellipse" src="./assets/Ellipse19.png" alt="">
+    </div>
+    <div class="skills">
+        <h6>Skills</h6>
+        <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+        </ul>
+
+    </div>
+    <hr>
+    <div class="social-icons">
+        <img src="./assets/git.png" alt="">
+        <img src="./assets/ln.png" alt="">
+        <img src="./assets/v.png" alt="">
+        <img src="./assets/tw.png" alt="">
+        <img src="./assets/m.png" alt="">
+    </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

Reply:
Done